.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,37 @@
+name: Build and test
+
+on:
+  push:
+    branches: ['**', '!backport/**']
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches: ['**']
+    paths-ignore:
+      - '**/*.md'
+
+jobs:
+  build-lint-test:
+    runs-on: ubuntu-latest
```

Comment:
Do we know that OUI builds on Windows? Should we add a `build and Verify` for Windows too?

Reply:
Added Windows workflow :+1: